blog/admin.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib import admin
+from blog.models import Category, Post, PostCategory, PostStatus, PostTag, Tag
+
+
```

Comment:
You can change this code to be more robust
`@admin.register(Post)
class PostAdmin(admin.ModelAdmin):
    list_display = ('title', 'slug', 'author', 'publish', 'status')
    list_filter = ('status', 'created', 'publish', 'author')
    search_fields = ('title', 'body')
    prepopulated_fields = {'slug': ('title',)}
    raw_id_fields = ('author',)
    date_hierarchy = 'publish'
    ordering = ('status', 'publish')`